xml/smb_native_os.xml

```diff
@@ -384,6 +384,33 @@
       <param pos="2" name="os.build"/>
    </fingerprint>
 
+      <!-- Mac OS X -->
+   <fingerprint pattern="^Samba (3\.0\.28a-apple)$">
```

Comment:
I can get both the point of including it or not. I guess we might want the version recored in the same way authenticated version check is done. If you run `smbd --version`,  the version string returned contains the `-apple`.